tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "outDir": "./static/js/dist",
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es5",
+    "jsx": "react",
+    "allowJs": true,
+    "moduleResolution": "node"
+  },
+  "exclude": [
```

Comment:
Instead of excluding, you should be able to only have it include ts files. That way you don't have to update it as you convert the files over. Simply changing the extension will be sufficient.

Reply:
OK, I've removed `exclude`. I tried adding `include` but it errors if it's empty but I'll add it once I start converting files.